docs/core/whats-new/dotnet-9/overview.md

```diff
@@ -59,7 +59,7 @@ ML.NET is an open-source, cross-platform framework that enables integration of c
 
 ## .NET Aspire
 
-.NET Aspire is an opinionated, cloud-ready stack for building observable, production ready, distributed applications.​ .NET Aspire is delivered through a collection of NuGet packages that handle specific cloud-native concerns, and is available in preview for .NET 9. For more information, see [.NET Aspire](/dotnet/aspire).
+.NET Aspire is a set of powerful tools, templates, and packages for building observable, production ready apps.​ .NET Aspire's latest release includes improvements to the dashboard and resource lifecycle management. It also adds new Integrations and APIs for more flexibility during your development. .NET Aspire 9 works with both .NET 9 and .NET 8 apps. For more information, see [.NET Aspire](/dotnet/aspire).
```

Comment:
We might want to point to the "What's new in .NET Aspire 9.0".

- `/dotnet/aspire/whats-new/dotnet-aspire-9`

Reply:
```suggestion
.NET Aspire is a set of powerful tools, templates, and packages for building observable, production ready apps.​ .NET Aspire's latest release includes improvements to the dashboard and resource lifecycle management. It also adds new Integrations and APIs for more flexibility during your development. .NET Aspire 9 works with both .NET 9 and .NET 8 apps. For more information, see [What's new in .NET Aspire 9](/dotnet/aspire/whats-new/dotnet-aspire-9).
```